### DISCUSSION TITLE
Scoping cookies to a path

### DISCUSSION THREAD

**Discussion by okarlsson** [ory/hydra#2822] [Q&A] [answered]
Hello 👋 

I have a scenario where hydra will be deployed on a specific path of a domain, so I would like the cookies that are set by hydra to be scoped to the same path. They are currently set to the path `/` and it does not seem to be configurable. 

The configuration reference only mentions the following config:
```
  cookies:
    same_site_legacy_workaround: true
    same_site_mode: Strict
```

Is there any way to set the path of the cookies?

**Comment by aeneasr** [comment, marked as answer]
Oh, that is in fact not configurable I believe at the moment. A PR to fix this would be appreciated! :)
